addons/web/static/tests/views/pivot_view_tests.js

```diff
@@ -5273,6 +5273,7 @@ QUnit.module("Views", (hooks) => {
     QUnit.test(
         "comparison with two groupbys: rows from reference period should be displayed",
         async function (assert) {
+            patchDate(2023, 2, 22, 1, 0, 0);
```

Comment:
We could fill the commit message to explain why we add a patchDate here. 🙂 

```txt
Before this commit, since new we are no longer in 2023, the test failed
because there is no longer 2021 in the filters dynamically added in the
dropdown of date filter. The reason is now we have current year, 
the previous one and the one before, that is, 2024, 2023, 2022.

This commit adds a patchDate to be sure we will start the test in 2023,
to get the expected filters displayed (2023, 2022, 2021).
```

Reply:
We will move that fix [in another PR](https://github.com/odoo/odoo/pull/147813) because the test also failed in 16.

See https://runbot.odoo.com/runbot/build/55822660